src/lib.rs

```diff
@@ -41,13 +42,15 @@ extern crate tempfile;
 extern crate zdaemon;
 extern crate zfilexfer;
 extern crate hostname;
+extern crate interfaces;
+extern crate ipnetwork;
 
 #[macro_use]
 mod ffi_helpers;
 mod command;
 mod project;
 mod directory;
-mod error;
+pub mod error;
```

Comment:
Why do you need to expose this module publicly? There's already a re-export of `Error` and `geterr`. I can't see what else in there needs to be public.